### ISSUE TITLE
Wrong timezone offset for past dates with milliseconds.

### ISSUE THREAD

**Issue by TomaszG** [moment/luxon#650]
It seems that returned offset for some past dates is not correct, for example:

1. 
```js
DateTime.fromJSDate(new Date('1920-01-01T00:00:00.001Z')).setZone('Europe/Warsaw').offset
```
returns `119.98333333333333`

2. 
```js
DateTime.fromJSDate(new Date('1930-01-01T00:00:00.001Z')).setZone('Europe/Warsaw').offset
``` 
returns `59.983333333333334`

3. 
```js
DateTime.fromJSDate(new Date('1500-01-01T00:00:00.001Z')).setZone('Europe/Warsaw').offset
``` 
returns `83.98333333333333`

while 
```js
DateTime.fromJSDate(new Date()).setZone('Europe/Warsaw').offset
```
correctly returns `60`.

**Comment by icambron** [comment]
Looks like there's an issue here with older dates with milliseconds in them. The core problem is that [this line](https://github.com/moment/luxon/blob/ac63fb1e6cb375abba1d6c995c665c3a2fd0938e/src/zones/IANAZone.js#L174) is assuming it should subtract to round down the date, but if the timestamp is negative, it should be adding.

**Comment by icambron** [comment]
Fixed in debf80b, will release shortly.
